sympy/physics/quantum/dagger.py

```diff
@@ -85,5 +85,13 @@ def __new__(cls, arg):
             return obj
         return Expr.__new__(cls, arg)
 
+    def __mul__(self, other):
+        from sympy.physics.quantum import IdentityOperator
```

Comment:
Imports should be at the top of the file.

Reply:
That leads to circular imports